primitives/Cargo.toml

```
@@ -11,6 +11,14 @@ repository.workspace = true
 [dependencies]
 sp-core = { workspace = true }
 substrate-prometheus-endpoint = { optional = true, workspace = true }
+num = { version = "0.4.0", default-features = false }
+num-rational = { version = "0.4.1", default_features = false, features = [
+    "num-bigint",
+] }
```

```suggestion
num = { version = "0.4.0", default-features = false, features = ["num-bigint"] }
```

`num` already re-exports the `num-rationals` crate under [num::rational](https://docs.rs/num/latest/num/rational/index.html)